config/ignition_citadel_gazebo11_ubuntu_focal.yaml

```diff
@@ -29,38 +42,67 @@ filter_formula: "\
  Package (= libignition-common3-graphics-dev) |\
  Package (= libignition-common3-profiler) |\
  Package (= libignition-common3-profiler-dev) \
-), $Version (% 3.14.0-1*) |\
+), $Version (% 3.14.2-1*) |\
 (Package (= ignition-fuel-tools4) |\
+ Package (= libgz-fuel-tools4) |\
+ Package (= libgz-fuel-tools4-dev) |\
  Package (= libignition-fuel-tools4) |\
  Package (= libignition-fuel-tools4-dev) \
-), $Version (% 4.4.0-1*) |\
+), $Version (% 4.6.0-1*) |\
 (Package (= ignition-gazebo3) |\
+ Package (= libgz-sim3) |\
+ Package (= libgz-sim3-dbg) |\
+ Package (= libgz-sim3-dev) |\
+ Package (= libgz-sim3-plugins) |\
  Package (= libignition-gazebo3) |\
  Package (= libignition-gazebo3-dbg) |\
  Package (= libignition-gazebo3-dev) |\
  Package (= libignition-gazebo3-plugins) \
-), $Version (% 3.13.0-1*) |\
+), $Version (% 3.14.0-1*) |\
 (Package (= ignition-gui3) |\
+ Package (= libgz-gui3) |\
+ Package (= libgz-gui3-dev) |\
  Package (= libignition-gui3) |\
  Package (= libignition-gui3-dev) \
-), $Version (% 3.9.0-1*) |\
+), $Version (% 3.11.2-1*) |\
 (Package (= ignition-launch2) |\
+ Package (= libgz-launch2) |\
+ Package (= libgz-launch2-dev) |\
  Package (= libignition-launch2) |\
  Package (= libignition-launch2-dev) \
-), $Version (% 2.2.2-1*) |\
+), $Version (% 2.3.0-1*) |\
 (Package (= ignition-math6) |\
+ Package (= libgz-math6) |\
+ Package (= libgz-math6-dbg) |\
+ Package (= libgz-math6-dev) |\
+ Package (= libgz-math6-eigen3-dev) |\
```

Comment:
```suggestion
 Package (= libgz-math6-eigen-dev) |\
```

According to the introspection performed by the generator script there is no package named `libgz-math6-eigen3-dev` but the `libignition-math6-eigen3-dev` package does use `eigen3` instead of `eigen`. @j-rivero do you know if this is a typo / omission or if the plan is to rename the libgz... package?

Reply:
I thought I had fixed that

https://github.com/gazebo-release/gz-math6-release/commit/3b8555972bd25344edbc3501cd9cec11a96f43d9

Reply:
yep, eigen3 should be used.

Reply:
The generator script is still finding the bare eigen package name on Focal. Looking at the package pool it appears that only jammy has the corrected package name.
![eigen](https://user-images.githubusercontent.com/358882/186238203-82f416e5-6821-4512-a06b-758ad3c9488a.png)